docs/guides/importing-organization.md

```diff
@@ -0,0 +1,154 @@
+---
+page_title: "Guide: Importing MongoDB Atlas Organizations"
+---
+
+# Importing MongoDB Atlas Organizations
+
+**Objective**: Learn how to import existing MongoDB Atlas organizations into Terraform management, understand why certain attributes are optional for imports, and successfully manage your existing organizations using Infrastructure as Code.
+
+## Overview
+
+Starting with version 1.38.0 of the MongoDB Atlas Provider, you can now import existing MongoDB Atlas organizations into your Terraform state. This feature allows you to bring organizations that were created outside of Terraform under Infrastructure as Code (IaC) management.
+
+## Why Import Organizations?
+
+Before version 1.38.0, the `mongodbatlas_organization` resource did not support the import functionality. Organizations could only be created through Terraform, which meant:
+
+- Existing organizations created through the Atlas UI or API couldn't be managed by Terraform.
+- Teams had to maintain organizations outside of their IaC workflows.
+- There was no migration path to bring existing organizations under Terraform management.
+
+With import support, you can now:
+- Bring existing organizations into Terraform management.
+- Standardize organization configuration across your infrastructure.
+- Apply consistent security settings and policies using IaC.
+
+## Important Changes to Required Attributes
+
+To enable import functionality, several attributes that were previously marked as required have been made optional:
+
+- `org_owner_id`
+- `description` 
+- `role_names`
+- `federation_settings_id` (was already optional but it's also creation-only)
+
+### Why These Changes Were Necessary
+
+These attributes were required attributes that:
+1. Are only needed when creating a new organization.
+2. Cannot be modified after the organization is created.
+3. Are not returned by the Atlas API when reading existing organizations.
+
+**Important:** While these attributes are now optional in the schema, they are still **required when creating a new organization**. The provider will validate that these attributes are present during the creation process.
+
+## How to Import an Organization
+
+### Prerequisites
+
+Before importing an organization, ensure you have:
+- MongoDB Atlas API credentials with Organization Owner permissions.
+- The Organization ID of the organization you want to import.
+- Terraform version 0.15 or later.
```

Comment:
Would not mention a fixed terraform version, we can reference our updated matrix if we want to call out a tf version to use